src/CosmosEditorManager.ts

```diff
@@ -17,12 +17,16 @@ import { MongoDocumentTreeItem } from './mongo/tree/MongoDocumentTreeItem';
 import { MongoDocumentNodeEditor } from './mongo/editors/MongoDocumentNodeEditor';
 import { MongoCollectionTreeItem } from './mongo/tree/MongoCollectionTreeItem';
 import { MongoCollectionNodeEditor } from './mongo/editors/MongoCollectionNodeEditor';
+import { DocDBStoredProcedureTreeItem } from './docdb/tree/DocDBStoredProcedureTreeItem';
+import { DocDBStoredProcedureNodeEditor } from './docdb/editors/DocDBStoredProcedureNodeEditor';
 
 export interface ICosmosEditor<T = {}> {
     label: string;
     id: string;
     getData(): Promise<T>;
     update(data: T): Promise<T>;
+    convertData(data: string): T;
```

Comment:
convertData is rather ambiguous.  Suggest convertFromString/convertToString
or even just: toString/fromString